docs/api-usage/authentication.mdx

```diff
@@ -642,7 +642,7 @@ Saleor Apps extending the dashboard will be passed a special JSON Web Token (JWT
 
 You can also use this token to authenticate browser calls to your application's API by validating the token in your backend. See the user authentication section for more details on obtaining the necessary JWKS.
 
-See the [Saleor SDK documentation](https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.md#events) for more details on how to receive the token.
+See the [Saleor SDK documentation](https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.mdx#events) for more details on how to receive the token.
```

Comment:
The link no longer exists. Should be moved to https://saleor-docs-git-adjustments-saleorcommerce.vercel.app/docs/3.x/developer/extending/apps/developing-apps/app-sdk/app-bridge#events